.vscode/tasks.json

```
@@ -96,6 +147,22 @@
       "options": {
         "cwd": "${workspaceFolder}"
       }
+    },
+    {
```

```suggestion
    {
    "type": "shell",
```
This is missing here

---

I don't think this is coming from my changes. Do we add it in this pull request anyway?

---

done